meta_package_manager/bitbar/meta_package_manager.7h.py

```diff
@@ -126,30 +151,36 @@ def print_menu():
         " ⚠️{}".format(total_errors) if total_errors else ""))
 
     # Print a full detailed section for each manager.
-    echo("---")
-    for manager in managers:
+    if recursive_menu:
+      submenu="--"
+    else:
+      submenu=""
 
+    if recursive_menu:
+        echo("---")
+    for manager in managers:
+        if not recursive_menu:
+            echo("---")
 
         if manager['error']:
-          # for line in manager['error'].strip().split("\n"):
-          #     echo("{:<23} {} | color=red font=Menlo size=10 trim=false emojize=false".format(manager.name + ':', line))
-          print_error(manager['error'])
+            print_error(manager['error'], manager['name'])
```

Comment:
Think I have solved this for now, by reverting my change of not printing the package info if an error occured. I have some ideas to to improve the errors a bit, but I think I'll save that to another PR if/when you have merged this one :-)